util/other.syms

```diff
@@ -396,6 +396,10 @@ OSSL_CMP_exec_IR_ses                    define
 OSSL_CMP_exec_CR_ses                    define
 OSSL_CMP_exec_P10CR_ses                 define
 OSSL_CMP_exec_KUR_ses                   define
+OSSL_CMP_CTX_get0_trusted               define
+OSSL_CMP_CTX_set0_trustedStore          define
```

Comment:
This is NOT a define

Reply:
Well, right. Removed `OSSL_CMP_CTX_set0_trustedStore` and `OSSL_CMP_CTX_get0_trustedStore` here.